victoriametrics-single.yaml

```diff
@@ -0,0 +1,56 @@
+package:
+  name: victoriametrics-single
```

Comment:
We could leave packages for single node configs just `victoriametrics` unless there are deviations in how these are named

Reply:
There are two differing naming conventions in their repo and docker image tags:
* In the [repo:](https://github.com/VictoriaMetrics/VictoriaMetrics/tags) tags are v1.102.1 and v1.102.1-cluster
* In [dockerhub](https://hub.docker.com/r/victoriametrics/vmagent/tags?page=&page_size=&ordering=&name=): they use `cluster` and `public-single-node`

Reply:
removed the `single` suffix